### ISSUE TITLE
 [Feature Request] Pause

### ISSUE THREAD

**Issue by starsoccer** [wesaphzt/privatelock#17]
Often times I may be playing a game or using an app that requires movement of my device. It would be convenient if I could pause private lock for a small period of time  like 10 mins while a game is being played.

**Comment by wesaphzt** [comment]
Hi, there is a pause feature in there already which you can access from the notification, and you can also adjust the pause length from the settings menu.

**Comment by starsoccer** [comment]
Hmm I hadnt actually realized that. It would be cool if you could add a note in the app as I couldnt find it.

It would also be useful if you could add pause as a quick action in the notification center. I didnt find this likely because I hide the notification since its always active and I dont want to see it.

**Comment by wesaphzt** [comment]
Yes, it's on my to-do list to expand the pause visibility as I'm aware some people hide the notification.

**Comment by starsoccer** [comment]
Okay great, another possible solution would be to just add a widget. Widget could allow for set pause intervals or turn the whole thing on/off

**Comment by wesaphzt** [comment]
There is a widget but it's limited to start/stop at the moment :) but adding it to the widget so you can cycle through all options is part of the plan.
